Homework/homework6/test_task2.py

```diff
@@ -0,0 +1,24 @@
+from homework6.task2 import instances_counter
+
+
+def test_get_created_instances():
+    @instances_counter
+    class Cls:
+        instances_counter = 42
+
+    Cls()
+
+    assert Cls.get_created_instances() == 1
+    assert Cls.instances_counter == 42
```

Comment:
Now you just test a random attribute.  Using `__instances_counter` as the name has more sense.